draft-ietf-mls-protocol.md

```diff
@@ -2158,7 +2413,8 @@ struct {
 struct {
   opaque epoch_secret<1..255>;
   opaque path_secret<1..255>;
-} GroupSecrets;
+  optional<PSKId> psk<1..2^32-1;
+} KeyPackage;
```

Comment:
There already is a struct called `KeyPackage`

Reply:
Hm. Not sure how that got there. I don't think we intended renaming the struct. The change was supposed to be that the `psk`s are included in the Welcome message.